Src/EmbeddedScripts.CSharp.MonoEvaluator/MonoCSharpRunner.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using EmbeddedScripts.CSharp.MonoEvaluator.Hosting;
+using EmbeddedScripts.CSharp.Shared.CodeGeneration;
+using EmbeddedScripts.Shared;
+using EmbeddedScripts.Shared.Exceptions;
+using Mono.CSharp;
+
+namespace EmbeddedScripts.CSharp.MonoEvaluator
+{
+    public class MonoCSharpRunner : ICodeRunner, IEvaluator
+    {
+        private StringWriter _writer = new StringWriter();
```

Comment:
nit: it's not generic writer, but errors and warnings writer, so can be renamed to smth more specific